src/index.html

```diff
@@ -15,13 +15,427 @@
+            <a
+              class="nav__link"
+              href="#0"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#0"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#0"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Image of APPLE A1419 iMac 27 Retina 5K Monoblock"
+        />
+
+        <div class="card__description">
+          <h2 class="card__description-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__description-code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="card__rate-stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__rate-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#0"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </article>
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Image of APPLE A1419 iMac 27 Retina 5K Monoblock"
+        />
+
+        <div class="card__description">
+          <h2 class="card__description-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__description-code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="card__rate-stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__rate-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#0"
+          class="card__button"
+          data-qa="hover"
```

Comment:
There is an inconsistency in the data-qa attribute value. In the previous card, it is 'card-hover', but here it is just 'hover'. Ensure consistency across similar elements unless there is a specific reason for the difference.